setup.py

```diff
@@ -16,18 +16,30 @@
 def read(fname):
     return open(os.path.join(os.path.dirname(__file__), fname)).read()
 
+def package_files(directory):
```

Comment:
it might be good to add an `extension` field so that only the 'xml' files are retrieved e.g.

```python
def package_files(directory, ends_with):
    paths = []
    for (path, directories, filenames) in os.walk(directory):
        for filename in filenames:
            if filename.endswith(ends_with):
                paths.append(os.path.join('..', path, filename))
    return paths
```